### PR TITLE
Fix argument escaping on Windows (issue #1)

### PR DESCRIPTION
**Pull request by JaniM**
Simple fix.

This was quite fun.

### CODE REVIEW

File: elvee.c
Comment (atifaziz) [inline elvee.c]: For consistency with the rest of the code format:

```suggestion
char *argv_quote(char *arg)
{
```

File: elvee.c
Comment (atifaziz) [inline elvee.c]: Indentation nitpick:

```suggestion
        argv[i] = argv_quote(argv[i]);
```

**Review by atifaziz — CHANGES_REQUESTED** [review]
Hey, thanks for this PR! If we can address the formatting nitpicks and address the leak then we are good to merge this!

**Comment by JaniM** [comment]
There we go. Sorry about that.

File: elvee.c
Comment (atifaziz) [inline elvee.c]: My detailed comment about addressing this leak somehow disappeared from [my last review](https://github.com/atifaziz/elvee/pull/2#pullrequestreview-367178805). Very odd. 😕

**Review by atifaziz — COMMENTED** [review]
@JaniM A comment I made in [my first review](https://github.com/atifaziz/elvee/pull/2#pullrequestreview-367178805) that [mysteriously never got posted](https://github.com/atifaziz/elvee/pull/2#discussion_r386834964) was that I'd like to address the memory leak from a code maintenance point-of-view. The comment about the leak was also something that belonged at the call site since it was potentially safe to ignore because the program ended shortly after. Anyway, I have addressed the leak but would appreciate another pair of eyes and a nod if all looks good to you too.

File: elvee.c
Comment (JaniM) [inline elvee.c]: "with with"

File: elvee.c
Comment (JaniM) [inline elvee.c]: I wonder, if the idea was to release rhe memory for code maintenance reasons, I assume there is a chance that you'll add code after this point? Is it wise to leave argv with dangling pointers here?

File: elvee.c
Comment (atifaziz) [reply elvee.c]: Thanks!

Comment (atifaziz) [reply elvee.c]: Addressed in 31b8545ca42f0f629743bc1bade0a4c4b60c4027.

File: elvee.c
Comment (atifaziz) [reply elvee.c]: That's a good point. I think it would be fair to leave a comment at this point (specifically) since it's a lot more obvious here that the program ends right after that a future maintainer would need to address should some post-processing be added. At the same time, it would be equally simple to just take care of it. Can I leave the decision with you and follow-up with the change? Thanks!

Comment (JaniM) [reply elvee.c]: There you go.

**Review by atifaziz — APPROVED** [review]
Briliant and thanks a lot for helping with the @JaniM. Enjoy your bounty. 💰

**Pull request merged**
